### ISSUE TITLE
Stop tracking time on sleep

### ISSUE THREAD

**Issue by karanvasudeva** [pkamenarsky/atea#22]
I'm not sure if this doesn't happen already: it would be nice to stop time-tracking for the duration that the computer is asleep, if it was on before going to sleep. Thanks!

**Comment by pkamenarsky** [comment]
No, currently this doesn't happen. Shouldn't be to hard, although if Java doesn't have an API for this this will have to be done natively in ObjC.

**Comment by jordelver** [comment]
It would also be great if any timer was stopped on exit. If I forget to stop Atea before turning off my machine the timer is still running when I turn the machine back on and I have to manually adjust the captured times :)

**Comment by franciscolourenco** [comment]
This is quite serious.. I think it should be marked as bug

**Comment by pkamenarsky** [comment]
Marked as bug.

**Comment by houshuang** [comment]
I wrote a very hackish Ruby time tracker (http://reganmian.net/blog/2013/03/29/time-tracker-one-week-on-new-features/), and added this feature using a separate "on-sleep" app. It also always starts tracking my time as "surfing" when I turn on the computer, does lots of visualization etc. I'm just learning Clojure, and very excited about this project - I might experiment with extending it and adding some of the functionality I had before (lots of global keyboard shortcuts, stats visualization etc).
